doc/modules/decomposition.rst

```diff
@@ -442,6 +455,9 @@ compensating for LSA's erroneous assumptions about textual data.
     <https://nlp.stanford.edu/IR-book/pdf/18lsi.pdf>`_
 
 
+|details-end|
```

Comment:
Can you please take the example (current lines 446 to 448) out of the dropdown? That may help keeping it visible.

Reply:
I've modified the PR. Please review.